azure-pipelines.yml

```diff
@@ -13,17 +16,55 @@ pr:
 jobs:
 - template: /eng/common/templates/jobs/jobs.yml
   parameters:
-    enableTelemetry: true
-    enablePublishBuildArtifacts: false
+    enableTelemetry: false
+    enablePublishBuildArtifacts: true
     helixRepo: dotnet/performance
     jobs:
-    - job: sdk_performance
-      pool:
-        name: Hosted VS2017
-      steps:
-      - checkout: self
-        clean: true
-      - script: echo "Hello World"
-        name: echo
-        displayName: Echo
-        condition: succeeded()
+      - job: sdk_performance_windows
+        timeoutInMinutes: 180
+        variables:
+        - ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
+          - name: _RunType
+            value: private
+          - name: BenchviewCommitName
+            value: '.NET Performance - CoreClr $(_RunType) - $(Build.SourceBranchName) $(System.PullRequest.PullRequestNumber)'
+        - ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+          - name: _RunType
+            value: rolling
+          - name: BenchviewCommitName
+            value: '.NET Performance - CoreClr $(_RunType) - $(Build.SourceBranchName) $(Build.SourceVersion)'
+        workspace:
+          clean: all
+        pool:
+          name: Hosted VS2017
+        strategy:
+          matrix:
+            x64_CoreCLR_netcoreapp30:
+              _Architecture: x64
+              _Category: CoreCLR
+              _Framework: netcoreapp3.0
+              _BuildConfig: $(_Architecture)_$(_Category)_$(_Framework)
+
+        steps:
+        - checkout: self
+          clean: true
+        - template: /eng/send-to-helix.yml
+          parameters:
+            HelixSource: 'pr/dotnet/performance/$(Build.SourceBranch)' # sources must start with pr/, official/, prodcon/, or agent/
+            HelixType: 'test/performance_$(_BuildConfig)/'
+            # HelixBuild: $(Build.BuildNumber) -- This property is set by default
+            HelixTargetQueues: Windows.10.Amd64.Open # Windows.Amd64.ClientRS4.DevEx.15.8.Perf # This will be whatever we call our perf pool
+            #HelixAccessToken: $(BotAccount-dotnet-github-anon-kaonashi-bot-helix-token)
+            #HelixPreCommands: eng\common\init-tools-native.cmd -InstallDirectory $(Agent.BuildDirectory)\native-tools -Force
+            # HelixPostCommands: '' -- any commands that you would like to run after running your job
+            WorkItemDirectory: $(Build.SourcesDirectory)\src\benchmarks\micro
+            WorkItemCommand: py -3 %HELIX_CORRELATION_PAYLOAD%\benchmarks_ci.py --benchview-submission-name "$(BenchviewCommitName)" --src-root %HELIX_WORKITEM_PAYLOAD% --bin-directory %HELIX_WORKITEM_PAYLOAD% --incremental no --category $(_Category) --architecture $(_Architecture) -f $(_Framework) --benchview-machinepool perfsnake --generate-benchview-data --benchview-run-type $(_RunType)  --bdn-arguments="--buildTimeout 300"
```

Comment:
`--bdn-arguments="--buildTimeout 300"` should not be needed after https://github.com/dotnet/performance/pull/191